package-lock.json

```diff
@@ -10,6 +10,7 @@
       "hasInstallScript": true,
       "license": "Apache-2.0",
       "workspaces": [
+        "api",
```

Comment:
This is curious. This line was removed in #4601. See discussion in #4621.

Reply:
Opened #4623 to fix this :slightly_smiling_face: